app/assets/stylesheets/kudos/components/_give-kudo.scss

```diff
@@ -3,8 +3,105 @@
   @extend .container;
   margin-bottom: $spacer;
   width: $give-kudo-width;
+  font-family: $font-gotham;
+  font-weight: 500;
+  text-align: center;
+}
+
+.give-kudo__modal {
+  text-align: left;
+  position: fixed;
+  z-index: 99 !important;
+  background: white;
+  border-radius: 4px;
+  padding: 35px 30px 35px 30px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.styled-kudo-button {
+  font-weight: 500;
+  font-size: 14pt;
+  color: $color-factual-orange;
+  background: transparent;
+  border-radius: 4px;
+  box-shadow:  $button-shadow;
+  border: none;
+  padding-right: 20px;
+  padding-left: 20px;
+
+  .fist-left {
+    padding-right: 5px;
+    transition: padding-right .1s linear 0s;
+  }
+
+  .fist-right {
+    padding-left: 5px;
+    padding-right: 15px;
+    transition: padding-left .1s linear 0s;
+  }
+}
+
+.styled-kudo-button:hover {
+  color: $hover-orange;
+  position: relative;
+  top: -1px;
+
+  .fist-left {
+    padding-right: 0;
+  }
+
+  .fist-right {
+    padding-left: 0;
+  }
+}
+
+.send-kudo-button {
+  height: 35px;
+  font-size: 12pt;
+}
+
+.open-modal {
+  height: 50px;
+}
+
+.close-modal {
+  padding-right: 20px;
+  padding-left: 20px;
+  height: 35px;
+  border-radius: 4px;
+  box-shadow:  $button-shadow;
+  font-weight: 500;
+  font-size: 12pt;
+  background: $color-factual-orange;
+  color: white;
+  border: none;
+}
+
+.close-modal:hover {
+  background: $hover-orange;
+  position: relative;
+  top: -1px;
+}
+
+.modal-button {
+  display: inline-block;
+  padding-left: 2.5px;
+  padding-right: 2.5px;
+}
+
+h3 {
+  font-size: 18pt;
+  font-weight: 600;
 }
 
+label {
+  font-weight: bolder;
```

Comment:
Can we use numbers for the font weights?

You can add some variables if you'd like to reference them by more semantic names (`$font-normal`, `$font-bold`, `$font-bolder`, etc).